components/content-service/pkg/layer/provider.go

```diff
@@ -227,6 +227,105 @@ func (s *Provider) GetContentLayer(ctx context.Context, owner, workspaceID strin
 	return nil, nil, xerrors.Errorf("no backup or valid initializer present")
 }
 
+// GetContentLayerPVC provides the content layer for a workspace that uses PVC feature
+func (s *Provider) GetContentLayerPVC(ctx context.Context, owner, workspaceID string, initializer *csapi.WorkspaceInitializer) (l []Layer, manifest *csapi.WorkspaceContentManifest, err error) {
+	span, ctx := tracing.FromContext(ctx, "GetContentLayer")
```

Comment:
Perhaps this is more correct?
```suggestion
	span, ctx := tracing.FromContext(ctx, "GetContentLayerPVC")
```